### PR TITLE
feature(triggers): add rolling upgrade trigger

### PR DESCRIPTION
**Pull request by fruch**
seem like it was forgotten, and was inside SCT code now it need to be update with trigger the centos9 job so introducing it into the code

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: jenkins-pipelines/master-triggers/sct_triggers/weekly-master-rolling-upgrades-trigger.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?><project>
+  <actions/>
+  <description>Triggers Master Tests on weekly basis every Saturday at 6:00&#13;
+&#13;
+UPDATE 02/02/2023: Removed rolling-upgrade-multi-dc-ami-test and rolling-upgrade-with-sla-no-shares-test + added the rolling-upgrade-azure-image-test under Roy's request&#13;
```
Comment (soyacz) [inline jenkins-pipelines/master-triggers/sct_triggers/weekly-master-rolling-upgrades-trigger.xml:5]: isn't it outdated? Does it matters?

**Review by soyacz — APPROVED** [review]
LGTM

File: jenkins-pipelines/master-triggers/sct_triggers/weekly-master-rolling-upgrades-trigger.xml (hunk 1 above)
Comment (fruch) [reply jenkins-pipelines/master-triggers/sct_triggers/weekly-master-rolling-upgrades-trigger.xml:5]: I don't know, that's the comments that was there, and I dumped it as is.

it's still true,  I.e. test mentioned are still removed, and the one added is still there.

**Review by vponomaryov — APPROVED** [review]
LGTM

**Pull request merged**